manifests/kustomize/third-party/mysql/base/mysql-deployment.yaml

```diff
@@ -42,6 +42,7 @@ spec:
         # https://dev.mysql.com/doc/refman/8.0/en/server-system-variables.html#sysvar_default_authentication_plugin
         # https://dev.mysql.com/doc/refman/8.0/en/server-system-variables.html#sysvar_authentication_policy
         - --default-authentication-plugin=mysql_native_password
+        - --binlog-expire-logs-seconds=864000
```

Comment:
NIT: should we add a comment here describing the settings and pointing to the docs: https://dev.mysql.com/doc/refman/8.0/en/replication-options-binary-log.html#sysvar_binlog_expire_logs_seconds ?

Reply:
Good suggestion, I'll add the comment.